### PR TITLE
Update Japanese translation

### PR DESCRIPTION
**Pull request by gemmaro**
Hello RubyGems,

This updates Japanese translation.
Each commit adds changes corresponding to those made to the respective English file (`en.yml`), as in the commit subject.

Thank you,
gemmaro.

- - -

TODOs to be ready for review.

- [x] Self-check the translation by looking at the way the messages are called.

### CODE REVIEW

**Review by simi — APPROVED** [review]
This is amazing. @hsbt can you review as well?

**Review by martinemde — COMMENTED** [review]
Nice!

I'll save approval for someone that can actually read Japanese. I really appreciate the help!

File: config/locales/ja.yml [hunk 1]
```diff
@@ -399,24 +399,24 @@ ja:
...
+      closing: こちらを申請しなかった場合はこのEメールはご放念ください。パスワードは変更されません。
+      opening: あなただと良いのですが、誰かがあなたにパスワードを変更するリンクを送るよう申請しました。
       title: パスワードを変更する
       subtitle: こんにちは、%{handle}！
   passwords:
     edit:
       submit: このパスワードを保存する
       title: パスワードをリセット
-      token_failure: URLを見返すか、再度送信してみてください。
+      token_failure: URLをダブルチェックするか、リセットする新しいパスワードを送ってみてください。
```
Comment (okuramasafumi) [inline config/locales/ja.yml:410]:
```suggestion
      token_failure: URLをダブルチェックするか、パスワードリセットを再度試してみてください。
```

File: config/locales/ja.yml [hunk 2]
```diff
@@ -399,24 +399,24 @@ ja:
...
     new:
       submit: パスワードをリセット
       title: パスワードを変更する
       will_email_notice: パスワード変更のためのリンクをEメールで送ります。
     create:
-      success:
-      failure_on_missing_email:
+      success: 数分でEメールが届きます。メールにはパスワードを変更する手順があります。
```
Comment (okuramasafumi) [inline config/locales/ja.yml:416]:
```suggestion
      success: 数分でEメールが届きます。メールにはパスワードを変更する手順が記載されています。
```

File: config/locales/ja.yml [hunk 3]
```diff
@@ -596,8 +596,10 @@ ja:
     show:
       title: "%{username} のプロフィール"
     security_events:
-      title:
-      description_html:
+      title: セキュリティ事象
+      description_html: |-
+        このページにはアカウントに起こったセキュリティ事象が表示されます。
```
Comment (okuramasafumi) [inline config/locales/ja.yml:601]:
```suggestion
        このページにはあなたのアカウントに起こったセキュリティ事象が表示されます。
```

File: config/locales/ja.yml [hunk 4]
```diff
@@ -726,7 +728,7 @@ ja:
       title: "%{name}の全バージョン履歴"
       versions_since:
         other: "%{since}からの%{count}件"
```
Comment (okuramasafumi) [inline config/locales/ja.yml:730]:
```suggestion
        other: "%{since}からの%{count}バージョン"
```

**Comment by gemmaro** [comment]
Hello @okuramasafumi, thank you for your suggestions.  I applied them.

**Review by martinemde — APPROVED** [review]
Thank you for your review, @okuramasafumi.

**Comment by okuramasafumi** [comment]
I wanted to approve this PR but GitHub didn't allow me, so I say "Great work!"

**Comment by simi** [comment]
Thanks all for nice co-operation in here.

**Comment by hsbt** [comment]
👍

**Pull request merged**